Sources/Apollo/ApolloStore.swift

```diff
@@ -324,13 +338,14 @@ public final class ApolloStore {
                        variables: GraphQLMap?) throws {
       let normalizer = GraphQLResultNormalizer()
       let executor = GraphQLExecutor { object, info in
-        return .result(.success(object[info.responseKeyForField]))
+        return .result(.success((object[info.responseKeyForField], Date())))
```

Comment:
I don't think we always want to update the `lastReceivedAt`with the current date for every write, because there's often a need to differentiate between server updates and (optimistic) client updates.